modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/IgniteH2Indexing.java

```diff
@@ -1413,57 +1410,9 @@ private Iterable<List<?>> executeSelect0(
      * @return Query results cursor.
      */
     private Iterable<List<?>> lockSelectedRows(Iterable<List<?>> cur, GridCacheContext cctx, int pageSize, long timeout) {
-        assert cctx != null && cctx.mvccEnabled();
```

Comment:
Do we need UnsupportedOperationException here? Otherwise, the behaviour variates depending on '-ea' JVM process option.

Reply:
We're not changing the behaviour, assert -> assert.
Just removing the unused code.
This method will be removed later as unused (when it become unused).